sycl/include/sycl/ext/xilinx/fpga/memory_properties.hpp

```diff
@@ -29,25 +33,46 @@ struct ddr_bank {
     }
   };
 };
+
+struct hbm_bank {
+  template <unsigned A> struct instance {
+    template <int B> constexpr bool operator==(const instance<B> &) const {
```

Comment:
Curious this mix of `int` and `unsigned`. Rationale?

Reply:
Rationale: duplication of existing code without much thinking.
Does not make sense...